src/pages/Upload/components/UploadInfo/Description/TextInput.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const TextInput = ({ textHandler }) => {
+  const [activeInput, setActiveInput] = useState(false);
+
+  const inputHandler = () => {
+    setActiveInput(!activeInput);
+  };
```

Comment:
```suggestion
  const inputHandler = () => {
    setActiveInput(prev => !prev);
  };
```
지금 상태도 크게 문제는 없지만 가급적 함수는 외부 값에 의존하지 않는 순수함수의 형태로 작성하려고 노력해주세요. 외부의 값을 가져와서 반대로 바꿔준다 보다는, 그냥 자기가 갖고 있는 값을 활용해 그것의 반대로 바꿔준다는 로직이 보다 나아보입니다.

Reply:
넵!! 세션때 설명해주신 내용인데 적용을 못했네요!! 앞으로 prev로 구성하겠습니다!!